Sources/include/ALMessage.h

```diff
@@ -26,6 +26,7 @@ static int const ALMESSAGE_CHANNEL_NOTIFICATION = 10;
 static int const ALMESSAGE_CONTENT_CUSTOM = 101;
 static int const ALMESSAGE_CONTENT_HIDDEN = 11;
 static int const ALMESSAGE_CONTENT_INITIAL_STATIC_MESSAGE = 102;
+static int const ALMESSAGE_CONTENT_TYPING_INDICATOR = 111;
```

Comment:
Did you cross check with backend team for the value

Reply:
verified with backend team we are not using 111 content type anywhere.